doc-generator.sc

```diff
@@ -117,33 +117,33 @@ def categoryAndSubcategoryOf(
   case "CKV_AWS_2" | "CKV_AWS_3" | "CKV_AWS_5" | "CKV_AWS_6" | "CKV_AWS_7" |
       "CKV_AWS_8" | "CKV_AWS_16" | "CKV_AWS_19" | "CKV_AWS_22" | "CKV_AWS_24" |
       "CKV_AWS_25" | "CKV_AWS_26" | "CKV_AWS_27" | "CKV_AWS_29" | "CKV_AWS_30" |
-      "CKV_AWS_31" | "CKV_AWS_32" | "CKV_AWS_33" | "CKV_AWS_34" | "CKV_AWS_38" |
```

Comment:
Some patterns already appeared in previous matches, so we had an unreachable code warning